R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -906,6 +910,24 @@ predicted <- predict(model, df)
 head(predicted)
 ```
 
+#### FP-growth
+
+`spark.fpGrowth` executes FP-growth algorithm to mine frequent itemsets on a `SparkDataFrame`.
+
+* `spark.freqItemsets` method can be used to retrieve a `SparkDataFrame` with the frequent itemsets.
+* `spark.associationRules` returns a `SparkDataFrame` with the association rules.
+
+
+```{r}
+items <- selectExpr(createDataFrame(data.frame(items = c(
+  "s,t,u",
```

Comment:
something that is not coded in 3 lines ;)
reading from a file if we could - if there isn't any dataset that we can license to use, can we anonymize an existing one?